Casks/c/copilot-cli.rb

```diff
@@ -13,6 +13,12 @@
   desc "Brings the power of Copilot coding agent directly to your terminal"
   homepage "https://docs.github.com/en/copilot/concepts/agents/about-copilot-cli"
 
+  # Use GitHub "latest release" when checking for new versions so prereleases are ignored.
```

Comment:
```suggestion
  # Some tagged versions using a stable version format are marked as
  # "pre-release" on GitHub, so it's necessary to check releases.
```

The `livecheck` block makes it clear that we're using the `GithubLatest` strategy, so let's focus on the "why" rather than the "how".

Reply:
I also renamed the branch that inadvertently closed this PR, so i reopened it here https://github.com/Homebrew/homebrew-cask/pull/240053